src/pipx/main.py

```diff
@@ -767,6 +815,20 @@ def setup_logging(verbose: bool) -> None:
     pipx_str = bold(green("pipx >")) if sys.stdout.isatty() else "pipx >"
     pipx.constants.pipx_log_file = setup_log_file()
 
+    # Determine logging level
+    if verbose >= 1:
+        level_number = logging.DEBUG
+    elif verbose == -1:
+        level_number = logging.WARNING
+    elif verbose == -2:
+        level_number = logging.ERROR
+    elif verbose <= -3:
+        level_number = logging.CRITICAL
+    else:
+        level_number = logging.INFO
```

Comment:
There is a level below DEBUG so these conditionals might need some tweaking. I’d probably do something like

```python
level = min(0, 2 - verbose) * 10
```

Reply:
Thanks, that simplifies it a lot! However, it only works without using the `min()` function, which always returns zero:

```python
level_number = (2 - verbose) * 10
```

Reply:
I meant max, sorry 😛 

The check is probably needed because otherwise `-vvvv` would return a nonsensical value.

Reply:
Hmm, all the verbose modes return `10` as a level number...

Reply:
Thanks, I applied your suggestion!